src/sortStudents.ts

```diff
@@ -1,7 +1,64 @@
-// describe Student type
-// create and export SortType enum
-// create SortOrder type
+interface Student {
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
+}
+export enum SortType {
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const sortedStudents: Student[] = [...students];
+  const reducer = (acc: number, curr: number): number => {
+    return acc + curr;
+  };
+
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      if (order === 'asc') {
```

Comment:
```suggestion
      if (order === 'asc') {
```

Use enum instead of string